source/client-applications.txt

```diff
@@ -22,7 +22,69 @@ Certified Partner Tools
 Connection Tutorials
 --------------------
 
-.. include:: /includes/toc/dfn-list-client-applications.rst
+You can connect to |bi-short| using any of these Business Intelligence
+tools.
+
+.. list-table::
```

Comment:
Try the stitch cards in a separate branch and PR, please spend no more than 30 mins on it though:

```
.. card-group::
   :type: large


   cards:
     - id: mongodb-queryanywhere
       headline: MongoDB QueryAnywhere
       subheadline: Use the MongoDB query language directly from your front-end code.
       image: /images/landing/configure-rules-based-access-to-mongodb.png
       link: https://docs.mongodb.com/stitch/getting-started/configure-rules-based-access-to-mongodb


     - id: serverless-functions
       headline: Serverless Functions
       subheadline: Run server-side code with access to app context and external services.
       image: /images/landing/integrate-third-party-services.png
       link: https://docs.mongodb.com/stitch/functions/


     - id: mongodb-mobile
       headline: MongoDB Mobile (Beta)
       subheadline: Leverage the full power of MongoDB embedded natively on mobile devices.
       image: /images/landing/mongodb-mobile.png
       link: https://docs.mongodb.com/stitch/mongodb/mobile/mobile-overview/


     - id: event-triggers
       headline: Event Triggers
       subheadline: Execute functions automatically in response to events in your application.
       image: /images/landing/create-server-side-functions-and-webhooks.png
       link: https://docs.mongodb.com/stitch/triggers/overview/
```

Reply:
Sounds good, Jon! Thanks.